tests/WE2E/end_to_end_tests.sh

```diff
@@ -3,44 +3,55 @@
 #----------------------------------------------------------------------
 #  Automation of UFS Short Range Weather App Worfklow End to End Tests.
 #
-#  The script is dependant on a successful build of this repo. 
+#  The script is dependant on a successful build of this repo using the
+#  test/build.sh script in the ufs-srweather-app repository.
 #  The UFS build must be completed in a particular manner for this 
 #  script to function properly, notably the location of the build and
 #  bin directories: 
-#    BUILD_DIR=${APP_DIR}/build_${compiler}  
-#    BIN_DIR=${APP_DIR}/bin_${compiler}  
-#  
+#    BUILD_DIR=${APP_DIR}/build_${compiler}
+#    BIN_DIR=${APP_DIR}/bin_${compiler}
+#
 #  With the creating of a test_log directory and output files describing
 #  the result of each workflow status, a testing framework is established.
 #  The user must verify the contents of each experiment file in test_log/ 
-#  to check workflow success or failure.   
+#  to check workflow success or failure.
 #
-#  Example: . end_to_end_tests.sh hera zrtrr
+#  Example: ./end_to_end_tests.sh hera zrtrr
```

Comment:
Here, I changed the instructions to execute instead of source this scripts. It's a bit safer.